.github/workflows/brokenlink-check.yml

```diff
@@ -0,0 +1,15 @@
+name: Links
+
+on:
+  workflow_dispatch:
+
+jobs:
+  my-broken-link-checker:
+    name: Check broken links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check
+        uses: ruzickap/action-my-broken-link-checker@v2
+        with:
+          url: https://qiskit.org
+          cmd_params: "--buffer-size=8192 --max-connections=10 --color=always --skip-tls-verification --exclude=\"(stubs|mylabs.dev|linkedin.com)\" --timeout=60"
```

Comment:
I believe this should work too:

```yml
cmd_params: --buffer-size=8192 \
            --max-connections=10 \
            --color=always \
            --skip-tls-verification \
            --exclude="(stubs|mylabs.dev|linkedin.com)" \
            --timeout=60
```

It will be more maintainable, e.g. if we want to change the `--exclude` list.

Reply:
✋🏼 learning here:

what is logic behind this line `stubs|mylabs.dev|linkedin.com`

I guess I don't know how they related to qiskit.org

Reply:
Hi! If a link contains any of these words, it will be excluded from the analysis (and the underlying links on those pages). 

mylabs.dev and linkedin.com are links from the example I started from. I kept them so as not to forget the notation.

Reply:
Oh, if they are not relevant to qiskit.org, then we should remove mylabs.dv and linkedin.com

Reply:
I tested the inline notation and it didn't work